gateway-server/server.py

```diff
@@ -0,0 +1,91 @@
+from flask import Flask, request, jsonify, render_template
+import requests
+# TODO: from kazoo.client import KazooClient
+from pymongo import MongoClient
+import random
+
+app = Flask(__name__)
+
+# TODO: Add Zookeeper functionality
+
+mongo_uri = '<hidden-for-pull-request>'
+client = MongoClient(mongo_uri)
```

Comment:
This mongo client is not distributed currently I guess? Consider firestore on GCP instead: https://cloud.google.com/firestore?hl=en